File: 1833-FindTheHighestAltitude/1833-FindTheHighestAltitude.java
// Last updated: 6/22/2026, 1:20:04 PM
class Solution {
    public int largestAltitude(int[] gain) {
        int maxi = 0;
        int val = 0;
        for(int i : gain){
            val = val + i;
            maxi = Math.max(maxi, val);
        }
        return maxi;
    }
}

File: 1833-FindTheHighestAltitude/1833-FindTheHighestAltitude.cpp
// Last updated: 6/22/2026, 1:20:02 PM
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int maxi = 0, val = 0;
        for(int i : gain){
            val += i;
            maxi = max(maxi, val);
        }
        return maxi;
    }
};